lib/api/src/sys/mod.rs

```diff
@@ -47,13 +47,13 @@ pub use crate::sys::types::{
 pub use crate::sys::types::{Val as Value, ValType as Type};
 pub use crate::sys::utils::is_wasm;
 pub use target_lexicon::{Architecture, CallingConvention, OperatingSystem, Triple, HOST};
-#[cfg(feature = "compiler")]
```

Comment:
We still want this line :)

Reply:
Reverting this breaks the `wasmer_api::wasmparser` import that you didn't want me to change previously?

Reply:
`wasmer/lib/c-api/src/wasm_c_api/unstable/parser/operator.rs` relies on `wasmer_api::wasmparser` but when building for headless the compiler flag isn't set and so the above mod file doesn't expose `wasmparser` which means this file won't compile. This issue is present in the current `main` branch, this fixes the existing issue.

It's either this or we add `wasmparser` as a dependency to `wasm_c_api`

Reply:
@syrusakbary - What do you think of the above?

Reply:
I've just fixed it